Gruntfile.js

```diff
@@ -48,12 +48,11 @@ module.exports = function( grunt ) {
 		];
 
 		browsers.ios = [
-			"bs_ios-5.1", "bs_ios-6.0", "bs_ios-7.0", "bs_ios-8.3",
-			"bs_ios-9.3", "bs_ios-10.3"
+			"bs_ios-10.3", "bs_ios-11.0", "bs_ios-11.2"
```

Comment:
This removes a lot of iOS versions and iOS simulators seem to be pretty stable compared to Android ones. Could you leave iOS as it was, only adding 11.2?